faucet/valve_of.py

```diff
@@ -630,7 +630,7 @@ def _match_ip_masked(ipa):
 def build_match_dict(in_port=None, vlan=None, eth_type=None, eth_src=None,
                      eth_dst=None, eth_dst_mask=None, icmpv6_type=None,
                      nw_proto=None, nw_dst=None, metadata=None,
-                     metadata_mask=None, vlan_pcp=None):
+                     metadata_mask=None, vlan_pcp=None, udp_src=None, udp_dst=None):
```

Comment:
@gizmoguy Is that too specific? Would you recommend a way of using a sort of nw_proto++.src or are you okay with that?